src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Declarations_SourceDeclarations.cs

```diff
@@ -57,7 +64,15 @@ public static Task<IEnumerable<ISymbol>> FindSourceDeclarationsAsync(Project pro
             }
         }
 
-        private static async Task<ImmutableArray<SymbolAndProjectId>> FindSourceDeclarationsWithNormalQueryAsync(
+        #endregion
+
+        #region Current API
+
+        // This region contains the current FindDeclaratins APIs.  The current APIs allow for OOP 
+        // implementation and will defer to the oop server if it is available.  If not, it will
+        // compute the results in process.
+
+        internal static async Task<ImmutableArray<SymbolAndProjectId>> FindSourceDeclarationsWithNormalQueryInLocalProcessAsync(
```

Comment:
I do not want to make these APIs public yet.  At the verry least, i'd want to make sure they were actually the right API to expose.  We're stuck in the position that the old APIs are public, and now we can't do anything about that.  I don't want to be in that position again.